src/WPF/ArcGISRuntime.WPF.Samples/Samples/Data/GenerateGeodatabase/GenerateGeodatabase.xaml

```diff
@@ -0,0 +1,20 @@
+﻿<UserControl x:Class="ArcGISRuntime.WPF.Samples.GenerateGeodatabase.GenerateGeodatabase"
```

Comment:
I don't like this UI for the WPF version (it works great for the others). I think maybe a border in the upper-left with the button and status bar (and maybe a descriptive label) would be better.